.github/workflows/ci-cd.yml

```diff
@@ -3,6 +3,9 @@ name: CI/CD Pipeline
 permissions: {}
 
 on:
+  schedule:
+    # run every day at 7:00 AM UTC
+    - cron: '0 7 * * *'
```

Comment:
This is nice, but I'm a bit concerned that we'd probably not get a nice notification if it fails